rebound/librebound.py

```diff
@@ -314,15 +315,48 @@ def save(self, filename):
     def load(self, filename):
         self.clibrebound.input_binary(c_char_p(filename.encode("ascii")))
         
+# Integrator Flags
+    @property 
+    def integrator_whfast_corrector(self):
+        return c_int.in_dll(self.clibrebound, "integrator_whfast_corrector").value
+    
+    @integrator_whfast_corrector.setter 
+    def integrator_whfast_corrector(self, value):
+        c_int.in_dll(self.clibrebound, "integrator_whfast_corrector").value = value
+
+    @property
+    def recalculate_jacobi_every_timestep(self):
+        return c_int.in_dll(self.clibrebound, "integrator_whfast_recalculate_jacobi_every_timestep").value
+
+    @recalculate_jacobi_every_timestep.setter
+    def recalculate_jacobi_every_timestep(self, value):
+        c_int.in_dll(self.clibrebound, "integrator_whfast_recalculate_jacobi_every_timestep").value = value
 
+    @property
+    def synchronize_every_timestep(self):
+        return c_int.in_dll(self.clibrebound, "integrator_whfast_synchronize_every_timestep").value
+
+    @synchronize_every_timestep.setter
+    def synchronize_every_timestep(self, value):
+        c_int.in_dll(self.clibrebound, "integrator_whfast_synchronize_every_timestep").value = value
+
+    @property
+    def particles_modified(self):
+        return c_int.in_dll(self.clibrebound, "integrator_whfast_particles_modified").value
+
+    @particles_modified.setter
+    def particles_modified(self, value):
+        c_int.in_dll(self.clibrebound, "integrator_whfast_particles_modified").value = value
+    
 # Integration
-    def step(self):
-        self.clibrebound.rebound_step()
 
-    def integrate(self, tmax, exactFinishTime=1, keepSynchronized=0, particlesModified=1, maxR=0., minD=0.):
+    def step(self, do_timing = 1):
+        self.clibrebound.rebound_step(c_int(do_timing))
+
+    def integrate(self, tmax, exact_finish_time=1, maxR=0., minD=0.):
         if debug.integrator_package =="REBOUND":
             self.clibrebound.integrate.restype = c_int
-            ret_value = self.clibrebound.integrate(c_double(tmax),c_int(exactFinishTime),c_int(keepSynchronized),c_int(particlesModified),c_double(maxR),c_double(minD))
+            ret_value = self.clibrebound.integrate(c_double(tmax),c_int(exact_finish_time),c_double(maxR),c_double(minD))
```

Comment:
spaces after commas in function arguments.  (sorry for the super minor comments! ... ignore them if you want obviously)